setup.cake

```diff
@@ -0,0 +1,106 @@
+
+var net46ArtifactPath = BuildParameters.Paths.Directories.PublishedApplications.Combine("Cake.Bakery/net46");
+var netCoreArtifactPath = BuildParameters.Paths.Directories.PublishedApplications.Combine("Cake.Bakery/netcoreapp1.1");
+
+Task("Copy-License")
+    .IsDependentOn("DotNetCore-Build")
+    .WithCriteria(() => BuildParameters.ShouldRunDotNetCorePack)
+    .Does(() =>
+{
+    // Copy license
+    CopyFileToDirectory("./LICENSE", net46ArtifactPath);
+    CopyFileToDirectory("./LICENSE", netCoreArtifactPath);
+});
+
+Task("DotNetCore-Pack-Custom")
+    .IsDependentOn("Copy-License")
+    .WithCriteria(() => BuildParameters.ShouldRunDotNetCorePack)
+    .Does(() =>
+{
+    var projects = GetFiles(BuildParameters.SourceDirectoryPath + "/**/*.csproj");
+    foreach(var project in projects)
+    {
+        var name = project.GetDirectory().FullPath;
+        if(name.EndsWith("Cake.Bakery") || name.EndsWith("Tests") || name.EndsWith("Cake.Scripting"))
+        {
+            continue;
+        }
+
+        DotNetCorePack(project.FullPath, new DotNetCorePackSettings {
+            NoBuild = true,
+            Configuration = BuildParameters.Configuration,
+            OutputDirectory = BuildParameters.Paths.Directories.NuGetPackages,
+            ArgumentCustomization = args => args
+                .Append("/p:Version={0}", BuildParameters.Version.SemVersion)
+                .Append("/p:AssemblyVersion={0}", BuildParameters.Version.Version)
+                .Append("/p:FileVersion={0}", BuildParameters.Version.Version)
+                .Append("/p:AssemblyInformationalVersion={0}", BuildParameters.Version.InformationalVersion)
+        });
+    }
+
+    var net46FullArtifactPath = MakeAbsolute(net46ArtifactPath).FullPath;
+    var net46FullArtifactPathLength = net46FullArtifactPath.Length+1;
+
+    // Cake.Bakery - .NET 4.6
+    NuGetPack("./nuspec/Cake.Bakery.nuspec", new NuGetPackSettings {
+        Version = BuildParameters.Version.SemVersion,
+        //ReleaseNotes = TODO,
+        BasePath = net46ArtifactPath,
+        OutputDirectory = BuildParameters.Paths.Directories.NuGetPackages,
+        Symbols = false,
+        NoPackageAnalysis = true,
+        Files = GetFiles(net46FullArtifactPath + "/**/*")
+                                .Select(file=>file.FullPath.Substring(net46FullArtifactPathLength))
+                                .Select(file=> !file.Equals("LICENSE") ? 
+                                    new NuSpecContent { Source = file, Target = "tools/" + file } :
+                                    new NuSpecContent { Source = file, Target = file })
+                                .ToArray()
+    });
+
+    var netCoreFullArtifactPath = MakeAbsolute(netCoreArtifactPath).FullPath;
+    var netCoreFullArtifactPathLength = netCoreFullArtifactPath.Length+1;
+
+    // Cake.Bakery - .NET Core
+    NuGetPack("./nuspec/Cake.Bakery.CoreCLR.nuspec", new NuGetPackSettings {

```

Comment:
Also due to the fact we have separate tool package for different target framework.

I would rather not specify `files` manually since it will introduce another possibility for errors. Also the amount of files are quite large.